src/Juvix/Compiler/Concrete/Pretty.hs

```diff
@@ -17,5 +18,11 @@ ppOutDefault = Print.ppOutNoComments defaultOptions
 ppOut :: (CanonicalProjection a Options, PrettyPrint c) => a -> c -> AnsiText
 ppOut = Print.ppOutNoComments
 
+ppPrint :: (PrettyPrint c) => c -> Text
+ppPrint = toAnsiText True . ppOutDefault
+
```

Comment:
I would remove this definition as it is not being used. Moreover, I think we should avoid having non-debug functions that render to Text with Ansi annotations. We should always print to AnsiText and let the renderer decide whether to use colors or not.